src/docs/resources/faq.md

```diff
@@ -884,11 +883,13 @@ We are focused on three things:
 ### Will Apple reject my Flutter app?
 
 We can't speak for Apple, but Apple's policies have changed over the
-years, and they have
-allowed apps built with systems like Flutter. We are aware of
-apps built with Flutter that have been reviewed and
-released via the App Store.
+years, and their App Store contains many apps built with technologies
+like Unity and Flutter.
+Apple has even featured Flutter apps such as [Hamilton.][Hamilton for iOS]
 
 Of course, Apple is ultimately in charge of their ecosystem, but our goal
-is to continue to do everything we can to ensure Flutter apps can be
+is to continue to do everything we can to ensure that Flutter apps can be
 deployed into Apple's App Store.
```

Comment:
deployed into -?-> deployed to

Reply:
"is to continue to do.." => "is continuing to do..."

And, yes, "to be deployed TO Apple's App Store."